[DOC-2781] Change ChaosHub to chaos hub (#994)
harness/developer-hub: docs/chaos-engineering/overview/Security/security-templates/kyverno-policies.md

title: Kyverno
sidebar_position: 3
---
[Kyverno policies](https://kyverno.io/policies/pod-security/) block configurations that don't match a policy (enforce mode) or can generate policy violations (audit mode). It scans existing configurations and reports violations in the cluster.

> HCE recommends using the provided policy configuration to enable the execution of all supported (out-of-the-box) experiments listed in the Enterprise chaos hub. Having said that, this is recommendatory in nature and left to user discretion or choice depending on the experiments desired.

Reviewer: ```suggestion
HCE recommends using the provided policy configuration to enable the execution of all supported (out-of-the-box) experiments listed in the Enterprise chaos hub. This is only a recommendation and left to user discretion depending on the experiments desired.
```
